src/dialogs/more-info/controls/more-info-light.js

```diff
@@ -291,6 +340,16 @@ class MoreInfoLight extends LocalizeMixin(EventsMixin(PolymerElement)) {
     });
   }
 
+  segmentClick() {
+    if (this.hueSegments === 24 && this.saturationSegments === 8) {
+      this.hueSegments = 0;
```

Comment:
Use `setProperties({ hueSegments: 0, saturationSegment: 0})` or else it will re-render twice

Reply:
Thanks, I just fixed this in a commit.